lib/addons/fluxcd/index.ts

```diff
@@ -0,0 +1,71 @@
+// lib/fluxcd_addon.ts
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FluxcdAddOnProps extends HelmAddOnUserProps {

```

Comment:
You don't want to add any of the properties specified [here](https://github.com/fluxcd-community/helm-charts/tree/main/charts/flux2)? Like multi-tenancy enabled, and such?

Ideally, I would like all values to be exposed in the addon in a strongly typed form. We can run `json2ts`  or https://marketplace.visualstudio.com/items?itemName=GregorBiswanger.json2ts to create a clear ts schema for the values.